tests/unittests/Makefile

```diff
@@ -253,6 +252,7 @@ INCLUDES += -I$(RIOTBASE)/tests/unittests/common
 # list of boards to run CI tests on
 TEST_ON_CI_WHITELIST += all
 
+include ../Makefile.tests_common
```

Comment:
I would keep the `unittest` Makefile untouched, it is a special one where we do not re-define `BOARD` anyway in the Makefile.
Also lot of things are done using `BOARD` before here.